frontend/src/app/about/page.tsx

```diff
@@ -159,20 +159,38 @@ const About = () => {
         </SecondaryCard>
 
         <SecondaryCard icon={faMapSigns} title={<AnchorTitle title="Roadmap" />}>
-          <ul>
-            {roadmap.map((item) => (
-              <li key={item.title} className="mb-4 flex flex-row items-center gap-2 pl-4 md:pl-6">
-                <div className="h-2 w-2 flex-shrink-0 rounded-full bg-gray-600 dark:bg-gray-300"></div>
-                <Link
-                  href={item.issueLink}
-                  target="_blank"
-                  className="text-gray-600 hover:underline dark:text-gray-300"
-                >
-                  {item.title}
-                </Link>
-              </li>
+          <div className="grid gap-4">
+            {projectMetadata.recentMilestones.map((milestone, index) => (
+              <div
+                key={index}
+                className={`flex items-center gap-4 overflow-hidden rounded-lg bg-gray-200 p-6 dark:bg-gray-700 ${
+                  index % 2 === 0 ? '' : 'ml-8'
+                }`}
+              >
+                <div className="flex-1">
+                  <Link
+                    href={milestone.url}
+                    target="_blank"
+                    className="inline-block hover:underline"
+                  >
+                    <h3 className="mb-2 text-xl font-semibold text-blue-400">{milestone.title}</h3>
```

Comment:
And instead of showing the progress bar we could add an icon here to any items that have a percentage (only if it's > 0) that will show that the milestone is in progress. 
I think something like `user-gear` [icon](https://fontawesome.com/icons/user-gear?f=classic&s=solid) should be good 👍🏼  Would be good to add a tooltip to it that will say something like `In progress` or `Started`